src/tools/ceph_objectstore_tool.cc

```diff
@@ -4690,9 +4706,22 @@ int main(int argc, char **argv)
         ret = clear_data_digest(fs.get(), coll, ghobj);
         goto out;
       } else if (objcmd == "clear-snapset") {
-        // UNDOCUMENTED: For testing zap SnapSet
-        // IGNORE extra args since not in usage anyway
-	if (!ghobj.hobj.has_snapset()) {
+        if (vm.count("arg1") == 0) {
```

Comment:
This is more than just documenting; the commit brings these validators as well.
Nothing wrong them them, this nit-picking is just about the commit's title.